src/loaders/load-typedefs.ts

```diff
@@ -97,12 +97,16 @@ export async function loadSingleFile(filePath: string, pluckOptions: ExtractOpti
   const extension = extname(filePath).toLowerCase();
   const fullPath = fixWindowsPath(isAbsolute(filePath) ? filePath : resolvePath(cwd, filePath));
 
-  if (extension === '.json') {
-    return loadFromJsonFile(fullPath);
-  } else if (GQL_EXTENSIONS.includes(extension)) {
-    return loadFromGqlFile(fullPath);
-  } else if (CODE_FILE_EXTENSIONS.includes(extension)) {
-    return loadFromCodeFile(fullPath, pluckOptions);
+  try {
+    if (extension === '.json') {
+      return await loadFromJsonFile(fullPath);
+    } else if (GQL_EXTENSIONS.includes(extension)) {
+      return await loadFromGqlFile(fullPath);
+    } else if (CODE_FILE_EXTENSIONS.includes(extension)) {
+      return await loadFromCodeFile(fullPath, pluckOptions);
+    }
+  } catch (e) {
+    throw new Error(`Failed to find any GraphQL type definitions in: ${filePath} - ${e.message}`);
```

Comment:
@kamilkisiela maybe we shouldn't throw an error here? just print it with `debugLog`. 
If an invalid file is loaded because of a broad glob (`src/**/*.ts`), we just want to warn about it and not to fail the entire scan.

Reply:
Related to: https://github.com/ardatan/graphql-toolkit/pull/174